Makefile

```diff
@@ -7,21 +10,29 @@ labels-config.json: \
 		./utility write_labels \
 		mid_level_categories > $@
 
+datasets/enwiki.labeled_wikiprojects.w_text.json: \
+		datasets/enwiki.labeled_wikiprojects.json
+	./utility fetch_text \
+		--api-host=https://en.wikipedia.org \
+		--labelings=$< \
+		--output=$@ \
+		--verbose
+
 datasets/enwiki.labeled_wikiprojects.w_cache.json: \
-	datasets/enwiki.labeled_wikiprojects.json
-	cat $< | \
-	revscoring extract \
-	drafttopic.feature_lists.wordvectors.drafttopic \
-	--host=https://en.wikipedia.org/ \
-	--input=datasets/enwiki.labeled_wikiprojects.json > $@
+		datasets/enwiki.labeled_wikiprojects.w_text.json
+	./utility extract_from_text \
+		drafttopic.feature_lists.wordvectors.drafttopic \
+		--input=$< \
+		--output=$@ \
+		--verbose
 
 models/enwiki.drafttopic.gradient_boosting.model: \
 	datasets/enwiki.labeled_wikiprojects.w_cache.json
 	cat $< | \
 	revscoring cv_train revscoring.scoring.models.GradientBoosting \
 		drafttopic.feature_lists.wordvectors.drafttopic mid_level_categories \
 		--debug \
-	   	--labels-config=labels-config.yaml \
+	   	--labels-config=labels-config.json \
```

Comment:
Seems like we should stick to yaml and rename the file.

Reply:
It's generated as json, though, so the fact that our --labels-config parser can understand the yaml superset is just a bonus.

Reply:
I guess, but in this case, it will expect yaml.  Hmm...  i guess I don't have that strong of an opinion.

Reply:
Me neither, I'm just lazy.  All I'm thinking is that the filename should follow its content, and that the accept-liberal param is akin to "libreoffice docfoo.xlsx", no reason to say "--labels-config-json" for exaple.